src/tests/ftest/erasurecode/restart.py

```diff
@@ -28,46 +55,136 @@ def execution(self, agg_check=None):
 
         Args:
             agg_check: When to check Aggregation status.Either before restarting all the servers or
-                       after.Default is None so not to check aggregation, but wait for 20 seconds
-                       and restart the servers.
+                       after. Default to None.
         """
-        # Write all EC object data to NVMe
-        self.ior_write_dataset(operation="Auto_Write", percent=self.percent)
-        self.log.info(self.pool.pool_percentage_used())
-        # Write all EC object data to SCM
-        self.ior_write_dataset(storage='SCM', operation="Auto_Write", percent=self.percent)
-        self.log.info(self.pool.pool_percentage_used())
-        self.pool.pool_percentage_used()
-
-        if not agg_check:
-            # Set time mode aggregation
-            self.pool.set_property("reclaim", "time")
-            # Aggregation will start in 20 seconds after it sets to time mode. So wait for 20
-            # seconds and restart all the servers.
-            time.sleep(20)
-
-        if agg_check == "Before":
-            # Verify if Aggregation is getting started
-            if not any(self.check_aggregation_status(attempt=50).values()):
-                self.fail("Aggregation failed to start Before server restart..")
-
-        # Shutdown the servers and restart
+        # 1.
+        ior_w_flags = self.params.get("flags", "/run/ior/iorflags/*")
+        ior_read_flags = self.params.get("read_flags", "/run/ior/iorflags/*")
+        block_transfer_sizes = self.params.get("block_transfer_sizes", "/run/ior/*")[0]
+        obj_class = self.params.get("dfs_oclass_list", "/run/ior/objectclass/*")
+        self.log_step("Create pool and containers")
+        self.pool = self.get_pool()
+        containers = []
+        for oclass in obj_class:
+            # Skip the object type if server count does not meet the minimum EC object
+            # server count
+            if oclass[1] > self.server_count:
+                continue
+            ec_object = get_data_parity_number(self.log, oclass[0])
+            containers.append(
+                self.get_container(
+                    self.pool, daos_command=self.get_daos_command(), oclass=oclass[0],
+                    properties="rd_fac:{}".format(ec_object["parity"])))
```

Comment:
FYI that `oclass_utils.extract_redundancy_factor` directly tells you the rd_fac of an object, and it works for more than just EC objects. For example:
```
from oclass_utils import extract_redundancy_factor
rd_fac = extract_redundancy_factor(oclass[0])
```